tenable/io/v3/vm/folders/api.py

```diff
@@ -79,20 +83,95 @@ def edit(self, id: int, name: str) -> None:
+                          SearchIterator,
+                          Response]:
         '''
-        Searches from the available folders.
+        Search and retrieve the folders based on supported conditions.
 
         :devportal:`folders: search <folders-search>`
 
+        Args:
+            fields (list, optional):
+                The list of field names to return from the Tenable API.
+                Example:
+                    - ['field1', 'field2']
+            filter (tuple, Dict, optional):
+                A nestable filter object detailing how to filter the results
+                down to the desired subset.
+                Examples:
+                    >>> ('or', ('and', ('test', 'oper', '1'),
+                    ...            ('test', 'oper', '2')
+                    ...     ),
+                    ... 'and', ('test', 'oper', 3)
+                    ... )
+                    >>> {'or': [
+                    ...         {'and': [
+                    ...                 {
+                    ...                     'value': '1',
+                    ...                     'operator': 'oper',
+                    ...                     'property': '1'
+                    ...                 },
+                    ...                 {
+                    ...                     'value': '2',
+                    ...                     'operator': 'oper',
+                    ...                     'property': '2'
+                    ...                 }
+                    ...             ]
+                    ...         }
+                    ...     ],
+                    ... 'and': [
+                    ...         {
+                    ...             'value': '3',
+                    ...             'operator': 'oper',
+                    ...             'property': 3
+                    ...         }
+                    ...     ]
+                    ... }
+                As the filters may change and sortable fields may change over
+                time, it's highly recommended that you look at the output of
+                the :py:meth:`tio.v3.vm.definitions.folders()`
+                endpoint to get more details.
+            sort list(tuple, Dict):
+                A list of dictionaries describing how to sort the data
+                that is to be returned.
+                Examples:
+                    - [('field_name_1', 'asc'),
+                             ('field_name_2', 'desc')]
+                    - [{'property': 'last_observed', 'order': 'desc'}]
+            limit (int):
+                Number of objects to be returned in each request.
+                Default is 200.
+            next (str):
+                The pagination token to use when requesting the next page of
+                results.  This token is presented in the previous response.
+            return_resp (bool):
+                If set to true, will override the default behavior to return
```

Comment:
Update docstring to describe correct behaviour.

Reply:
Updated